### DISCUSSION TITLE
Question about --var-mods flag

### DISCUSSION THREAD

**Discussion by Munchic** [vdemichev/DiaNN#436] [Q&A]
Hi,

I wanted to clarify, does the --var-mods parameter specify the maximum of all variable modifications together or the maximum for each specified modification? E.g., if I set UniMod:21 and UniMod:35 and var-mods=1, can they co-occur on a modified peptide?

Thank you.

**Comment by Munchic** [comment]
It looks like it would set the maximum of variable modifications on a single peptide.

**Reply by vdemichev** [reply]
Yes, exactly.
